cypress/integration/email/mention_email_notification_spec.js

```diff
@@ -12,17 +12,23 @@
 import users from '../../fixtures/users.json';
 import * as TIMEOUTS from '../../fixtures/timeouts';
 
-import {reUrl} from '../../utils';
+import {getEmailUrl, getEmailMessageSeparator, reUrl} from '../../utils';
 
 const user1 = users['user-1'];
 const user2 = users['user-2'];
 
 const text = `Hello @${user2.username}`;
-const feedbackEmail = 'test@example.com';
+let config;
 
 describe('Email notification', () => {
     before(() => {
-        cy.apiUpdateConfig({EmailSettings: {FeedbackEmail: feedbackEmail}});
+        cy.apiGetConfig().then((response) => {
+            config = response.body;
+        });
+
+        cy.apiLogin('user-2');
+        cy.apiUpdateUserStatus('offline');
```

Comment:
@saturninoabril Is setting user-2 to be offline because user-2 could be logged-in in another test? If so, maybe creating a new user to get the email would be better?

Reply:
Yes.  That's good point, will do that, thanks

Reply:
@thekiiingbob Updated, please take a look again, thanks